src/app.ts

```diff
@@ -18,7 +18,7 @@ export class App {
 
   constructor(controller: Controller[]) {
     this.app = express();
-    this.port = process.env.PORT || 3000;
+    this.port = process.env.PORT || 5000;
```

Comment:
Minor: Question: Curious why we are exposing this port instead of the 80.

Reply:
In NodeJS world usually are used `80`, `3000`, `5000` and `8080`. We can go with 80, no problem.

Reply:
I think `80` makes sense for public APIs, otherwise it doesn't matter